operator/pkg/operator.go

```diff
@@ -154,6 +212,9 @@ func (o *Operator) Start(ctx context.Context) error {
 		metricsErrChan = make(chan error, 1)
 	}
 
+	batchProcessorChan := make(chan uint32)
```

Comment:
This channel is to mark a batch as processed, right?
```suggestion
	batchProcessedChan := make(chan uint32)
```